features/payment-request.yml

```
@@ -0,0 +1,41 @@
+name: Payment Request
+description: The Payment Request API allows the browser to offer a consistent and streamlined user experience while performing financial transactions.
```

- ditto on the `API` bit
- removed a passive voice
- simplified
```suggestion
description: The Payment Request API allows the browser to offer a consistent user experience for making payments.
```

Here also, maybe worth summarizing the components and how they interact, eg a big chunk of the API surface here is about working with payment responses.